config/src/config.rs

```diff
@@ -276,6 +281,17 @@ pub struct Mining {
     pub enabled: bool,
 }
 
+/// Ntp-related configuration
+#[derive(PartialStruct, Debug, Clone, PartialEq)]
+#[partial_struct(derive(Deserialize, Default, Debug, Clone, PartialEq))]
+pub struct Ntp {
+    /// Period that indicate the validity of a ntp timestamp
+    pub update_period: i64,
```

Comment:
Maybe change this field from i64 to Duration, for consistency? But I see that there are other fields which represent a time duration but are i64s...

    #[partial_struct(serde(
        default,
        deserialize_with = "from_secs",
        rename = "feeler_peers_period_seconds"
    ))]
    pub feeler_peers_period: Duration,